specification/snippets/CorrelationHeader.yaml

```diff
@@ -1,4 +1,4 @@
 schema:
   type: string
   pattern: 11C46F5F-CDEF-4865-94B2-0EE0EDCC26DA
-description: 'The X-Correlation-ID from the request header, if supplied, mirrored back.'
\ No newline at end of file
+  description: 'If the X-Correlation-ID header was included in the request the value of this header will be equal to that. Otherwise it will be a randomly generated alphanumeric string.'
```

Comment:
Is it worth mentioning that it will be the `apigee message id`, rather than a `randomly generated alphanumeric string`?

Reply:
I don't think so, users really shouldn't know what tech stack we're using as it only open vulnerabilities. Its the same reason we strip out Amazon headers

Reply:
There's no hiding the fact that we are using apigee.
As it currently stands it's not accurate. It's the `randomly generated` part. How about `Otherwise it will be a unique alphanumeric string generated by NHS Notify`?

Reply:
Well we should be hiding the fact we use apigee.

Your wording is much better.

Reply:
The string apigee features 399 times in the repo :-)